src/SFML/Graphics/Image.cpp

```diff
@@ -212,11 +213,43 @@ bool Image::loadFromFile(const std::filesystem::path& filename)
     // Clear the array (just in case)
     m_pixels.clear();
 
+    std::ifstream file(filename, std::ios::binary);
+    if (!file.is_open())
+    {
+
```

Comment:
Unwanted empty line

Reply:
That's true, I'll try to not forget to remove it next time we make changes to this PR.